packages/lab/src/tabs-next/TabNext.css

```diff
@@ -0,0 +1,72 @@
+/* Class applied to root Tab element */
+.saltTabNext {
+  align-items: center;
+  justify-content: center;
+  appearance: none;
+  display: inline-flex;
+  background: var(--salt-navigable-primary-background);
+  border: 0 solid transparent;
+  border-radius: 0;
+  white-space: pre;
+  min-width: 4em;
```

Comment:
what's the rule on magic numbers?

Reply:
I think this one is fine. It's a shared value between Input and Tabs